lib/rspec_parameterized_context.rb

```diff
@@ -52,7 +52,16 @@ def define_spec
     end
 
     def method_missing(action, *args, **options, &block)
-      @rspec_context.send(action, *args, **options, &block)
+      if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7.0')
+        @rspec_context.send(action, *args, **options, &block)
+      else
+        if options&.empty?
+          # calling double splat operator for empty hash returns `{}` on before Ruby 2.7 but we prefers returning nil
+          @rspec_context.send(action, *args, &block)
```

Comment:
@alpaca-tc 
Calling double splat operator for empty hash returns `nil` on after ruby2.7. But before the ruby version double splat operator for empty hash returns `{}`, so some test failed.
Let me know your opinion because I'm not sure these changes are right ><